baybe/surrogates/bandits/base.py

```diff
@@ -0,0 +1,57 @@
+"""Multi-armed bandit surrogates."""
+
+from __future__ import annotations
+
+from attrs import define, field
+from attrs.validators import instance_of
+from pandas import DataFrame
+
+from baybe.objectives.base import Objective
+from baybe.objectives.single import SingleTargetObjective
+from baybe.priors.base import Prior
+from baybe.searchspace.core import SearchSpace
+from baybe.surrogates.bandits.beta_bernoulli import (
+    BetaBernoulliMultiArmedBanditSurrogate,
+)
+from baybe.surrogates.base import Surrogate
+from baybe.targets.binary import BinaryTarget
+
+
+def _get_bandit_class(
+    searchspace: SearchSpace, objective: Objective
+) -> type[BetaBernoulliMultiArmedBanditSurrogate]:
+    """Retrieve the appropriate bandit class for the given modelling context."""
+    match searchspace, objective:
```

Comment:
included the searchspace in this private (so not affected by deprecation considerations) function why?